src/pkg/packaging/rpm/package.targets

```diff
@@ -0,0 +1,271 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Import Project='package.props' />
+
+  <UsingTask TaskName="ReplaceFileContents" AssemblyFile="$(LocalBuildToolsTaskDir)core-setup.tasks.dll"/>
+  <UsingTask TaskName="BuildFPMToolPreReqs" AssemblyFile="$(LocalBuildToolsTaskDir)core-setup.tasks.dll"/>
+
+  <PropertyGroup>
+    <IsRPMBasedDistro Condition="$(PackageTargetRid.StartsWith('rhel')) or
```

Comment:
For this round we are keeping this scoped to RHEL and Centos. In next round we can expand to Fedora and Opensuse once we have done more testing. 

We still do not have a generic file name for the rpms -- so that is another thing to resolve before we extend.